Release/2.1.0

sche opened the pull request.

valeriyvan commented:
I have missed this.
Yesterday read some discussion it might be changed in future.

DmitryBespalov commented:
won't fix until new Swift arrives.

Commit:
refs/heads/release/2.1.0 update marketing version

Commit:
gh-549 disable opening of transaction details from a push notificaion

Commit:
Merge pull request #554 from gnosis/gh-549-nav-bar-issue

gh-549 disable opening of transaction details from a push notificaion

Commit:
Merge branch 'main' into release/2.1.0

Commit:
refs/heads/release/2.1.0 Merge branch 'main' of github.com:gnosis/safe-ios into release/2.1.0

# Conflicts:
#	Multisig/UI/Transactions/TransactionDetails/TransactionDetailsView.swift
#	Multisig/UI/Transactions/TransactionsList/TransactionListView.swift
#	Multisig/UI/Transactions/TransactionsList/TransactionsView.swift
